.github/workflows/tests.yaml

```diff
@@ -34,21 +34,38 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Load secrets
-        working-directory: tests/functional_tests
+        working-directory: tests/functional_tests/tools/virtualenv/keys
         run: |
           mkdir -p ~/.atsign/keys/
-          echo "${{ secrets.ATKEYS_12ALPACA }}" > ~/.atsign/keys/@12alpaca_key.atKeys
-          echo "${{ secrets.ATKEYS_12SNOWBOATING }}" > ~/.atsign/keys/@12snowboating_key.atKeys
+          ls -la
+          cp "@alice🛠_key.atKeys" ~/.atsign/keys/
+          cp "@bob🛠_key.atKeys" ~/.atsign/keys/
+
+      - name: Start Virtual Environment
+        working-directory: tests/functional_tests/tools/virtualenv
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y docker-compose
+          sudo ./start_virtualenv.sh
+          sleep 5
+          sudo ./pkam_virtualenv.sh
+          sleep 1
+
+      - name: Add vip.ve.atsign.zone to /etc/hosts
+        run: |
+          echo "127.0.0.1 vip.ve.atsign.zone" | sudo tee -a /etc/hosts
+          cat /etc/hosts
```

Comment:
More debug stuff

Reply:
Done in https://github.com/atsign-foundation/at_c/pull/476/commits/e91dc26f51097b190c93ec3f815b6f421e0fe5a6